### ISSUE TITLE
Little NavigationView Stuff

### ISSUE THREAD

**Issue by nlogozzo** [amwx/FluentAvalonia#76]
Two Things:
![image](https://user-images.githubusercontent.com/17648453/150683291-13890708-0f1f-476e-acd3-c2adf16dcc0b.png)

1. Should there be this big left padding in the first navigation view item (Home)?
2. Why is there is slight line border between the navigation view and the frame?

For reference, this is how my NavView is designed:
```xml
<ui:NavigationView PaneDisplayMode="Top" IsSettingsVisible="False" SelectedItem="{Binding SelectedNavItem}" Header="{Binding NavViewTitle}">
    <ui:NavigationView.MenuItems>
        <ui:NavigationViewItem Content="Home" Tag="Home" Icon="Home" IsSelected="True"/>
    </ui:NavigationView.MenuItems>
    
    <ui:NavigationView.FooterMenuItems>
        <ui:NavigationViewItem Content="Settings" Tag="Settings" Icon="Settings"/>
    </ui:NavigationView.FooterMenuItems>
    
    <Grid>
        <ui:Frame HorizontalAlignment="Stretch" VerticalAlignment="Stretch" Content="{Binding SelectedPage}"/>
        
        <ui:InfoBar Name="InfoBar" HorizontalAlignment="Stretch" VerticalAlignment="Bottom"/>
    </Grid>
</ui:NavigationView>
```

**Comment by amwx** [comment]
1 is there because that's where the back button is. Technically no, that space shouldn't be there if the button isn't visible. But IIRC when I was porting the template, Avalonia's styling system isn't robust enough to handle what needs to be done there, which means I'd have to do it in code - which breaks if anybody tries to restyle the NavigationView. So for now, it gets the cliche it's a feature not a bug

Not sure why 2 is there. There had to have been a border at some point which is why its in my code, but I don't see it in WinUI's source anymore.

**Comment by nlogozzo** [comment]
Is there a way I can manually fix #1 in my codebase?

**Comment by amwx** [comment]
After having a look via dev tools, it turns out it's not what I said above (it's been so long since I've looked at this I've forgotten I found a way around that). Turns out there's a rogue `ContentControl` in the template (or something) that's sizing to 48px width and it will not respond to anything to get it to not do that. So no, for now, you can't override this, because I have no idea why this is happening. There's nothing in the code, no Style Selectors, and no width setting on the actual element so it seems the Navigation View has started practicing black magic. Am still investigating...

**Comment by nlogozzo** [comment]
Uh oh....black magic 😨 
Haha no worries!

**Comment by amwx** [comment]
# 2 should be fixed now in v1.2. The first one still remains a mystery
